pkgpanda/docker/dcos-builder/Dockerfile

```diff
@@ -50,9 +50,9 @@ RUN ln -sf /usr/bin/cpp-4.8 /usr/bin/cpp && \
   ln -sf /usr/bin/gcc-ranlib-4.8 /usr/bin/gcc-ranlib && \
   ln -sf /usr/bin/gcov-4.8 /usr/bin/gcov
 
-ENV GOLANG_VERSION 1.8.3
```

Comment:
I'm not sure we can upgrade docker just like that for DCOS :D

Reply:
@nLight What would you suggest?

Reply:
I suggest we ship pre-built binary instead. I bet @Poltergeist and @DanielMSchmidt are working on that

Reply:
Okay, we will add it here once it is ready.

Reply:
@GeorgiSTodorov The release setup is "done", at least you could take a look at @Poltergeist PR, the upload is implemented in that jenkinsfile

Reply:
@DanielMSchmidt Thank you, is ` sh "aws s3 cp build/dcos-ui-update-service s3://downloads.mesosphere.io/dcos-ui-update-service/latest/dcos-ui-update-service"` what I need here?

Reply:
@GeorgiSTodorov Please write me in slack next time, as the repo you copied from is private and this one is not. But yeah, the download url is https://downloads.mesosphere.com/dcos-ui-update-service/latest/dcos-ui-update-service

Reply:
@DanielMSchmidt Okay, I deleted my previous comment.

Reply:
Let's please revert the changes in the file